v21.1/set-locality.md

```diff
@@ -64,6 +64,8 @@ For more information about how table localities work, see [Regional tables](mult
 
 ### Set the table locality to `REGIONAL BY ROW`
 
+{{site.data.alerts.callout_info}} Changefeeds are not currently supported on <em>regional by row</em> tables. When a table's locality is set to `REGIONAL BY ROW`, changefeed jobs targeting that that table will fail. {{site.data.alerts.end}}
```

Comment:
Hi Abbey!  This content LGTM except there is a rendering issue in this callout which I think will be solved by editing the format slightly to the below (also I snuck in a link to CDC stuff, hope that's OK):

```
{{site.data.alerts.callout_info}}
[Changefeeds](stream-data-out-of-cockroachdb-using-changefeeds.md) are not currently supported on regional by row tables. When a table's locality is set to `REGIONAL BY ROW`, any changefeed jobs targeting that that table will fail.
{{site.data.alerts.end}}
```

Alternatively, I can fix this in a follow-up PR if you need to move on to something else, I know this part is not your day job :-)